docker/Dockerfile_centos_77

```diff
@@ -35,3 +35,10 @@ RUN mkdir /llvm-project-11.1.0.src/build && \
     make install -j 8 && \
     rm -rf /llvm-project-11.1.0.src
 
+RUN git clone https://github.com/flink-extended/clink.git /root/clink && \
+    cd /root/clink && \
+    git submodule update --init --recursive && \
+    bazel build --disk_cache=~/.cache/bazel @tf_runtime//tools:bef_executor_lite && \
```

Comment:
Is it necessary to add the `--disk_cache=~/.cache/bazel`? Do you know why the `bazel build` in the README won't need this flag?

Reply:
As discussed offline, I did not find detailed documentations about Bazel's behavior when `disk_cache` is not set. It has only been experimentally proved that `--disk_cache=~/.cache/bazel` must be add in Dockerfile and can optionally be add in commands shown in README. As a deep look into this issue requires certain workload and the issue might not worth this workload so far as I can see, let's reserve it for now and maybe we can re-visit it when we have time or it turns out to be a problem.